### PR TITLE
Add placeholder question mark for 2018–present in `History of osu!`

### PR DESCRIPTION
**Pull request by tematikkp**
This commit is meant to be discussed as this is an outdated page.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/en.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Records of osu!'s history, all in the osu! wiki.
 - [2016](2016) - scorev2, announcement of a new open-source "lazer" version of osu! (previously known as "osu!lazer"), and an osu! wiki overhaul
 - [2017](2017) - release of the first lazer build of osu!, and Discord integration on the stable client
 - [2018](2018) - ?
+- [2019](2019) - ?
+- [2020](2020) - ?
+- [2021](2021) - ?
+- [2022](2022) - 15 years of osu!, ?
+- [2023](2023) - ?
```
Comment (TicClick) [inline wiki/History_of_osu!/en.md:26]: if you are modelling this after 2018, consider that it at least has a dedicated page, while the years you have added links for don't, so it's better if you do it this way:
```suggestion
- 2019–2023 - ?
```

if you regret losing the 15 years remark, don't -- it's an event significant enough, so people will remember and write it down once they get to work on the 2022 article

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/History_of_osu!/en.md:26]: alternative option: `2019–present - ?`

Comment (tematikkp) [reply wiki/History_of_osu!/en.md:26]: honestly, I agree with this changes, but 2018 has also a question mark

Comment (TicClick) [inline wiki/History_of_osu!/en.md:26]: could also do this
```suggestion
- [2018](2018)–present - ?
```

Comment (TicClick) [reply wiki/History_of_osu!/en.md:26]: see my comment below (it changes the 2018 line as well, as opposed to the suggestion in this comment thread)

**Comment by Walavouchey** [comment]
@TicClick @Pisapou @respektive quick check on translations if you don't mind ty

french one was copied from https://osu.ppy.sh/wiki/fr/History_of_osu%21/Online_rankings/osu%21#2021

File: wiki/History_of_osu!/de.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Aufzeichnungen der Geschichte von osu!, alle im osu!-Wiki.
 - [2015](2015) - jizz (neu gestaltetes osu!web), osu!weekly und osu!coins
 - [2016](2016) - scorev2, Ankündigung einer neuen quelloffenen Version von osu! namens "lazer" (vorher bekannt als "osu!lazer") und eine Überarbeitung des osu!-Wikis
 - [2017](2017) - erste Veröffentlichung des Builds lazer von osu! und Discord-Integration auf dem Stable-Client
-- [2018](2018) - ?
+- [2018](2018)–Gegenwart - ?
```
Comment (respektive) [inline wiki/History_of_osu!/de.md:20]:
```suggestion
- [2018](2018)–Heute - ?
```

**Review by respektive — CHANGES_REQUESTED** [review]

**Review by Pisapou — APPROVED** [review]
french is ok

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**